sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Microsoft.Azure.WebJobs.Extensions.ServiceBus.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
     <Description>Microsoft Azure WebJobs SDK ServiceBus Extension</Description>
-    <Version>5.14.0-beta.1</Version>
+    <Version>5.13.3</Version>
```

Comment:
Was this intentional?   If so, don't we need to kill the project ref below on L57?

Reply:
Yeah I'm going to ship today. Removed the project ref.